sale_cancel_reason/wizard/cancel_reason_view.xml

```diff
@@ -1,38 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-    <data noupdate="0">
-        <record id="view_sale_order_cancel" model="ir.ui.view">
-            <field name="name">Reason for the cancellation</field>
-            <field name="model">sale.order.cancel</field>
-            <field name="arch" type="xml">
-             <form string="Reason for the cancellation" version="7.0">
-                <p class="oe_grey">
-                    Choose the reason for the cancellation of the
-                    sale order.
-                </p>
-                <group>
-                    <field name="reason_id" widget="selection"/>
-                </group>
-                <footer>
-                    <button name="confirm_cancel"
-                        string="Confirm" type="object"
-                        class="oe_highlight"/>
-                    or
-                    <button string="Cancel" class="oe_link"
-                        special="cancel" />
-                </footer>
-            </form>
-            </field>
-        </record>
+<data noupdate="0">
```

Comment:
@JayVora-SerpentCS would you be able to fix this misunderstanding? Like @pedrobaeza says, the *data* tag is redundant in 9.0, so please replace with the *odoo* tag.